src/sentry/tasks/post_process.py

```diff
@@ -983,12 +987,14 @@ def process_rules(job: PostProcessJob) -> None:
         # TODO(dcramer): ideally this would fanout, but serializing giant
         # objects back and forth isn't super efficient
         callback_and_futures = rp.apply()
+
+        # TODO(cathy): add opposite of the FF organizations:workflow-engine-issue-alert-fire-actions
```

Comment:
is this for when we only want to have 1 processor enabled? (we could just exit at the beginning of the method in those cases; `if not features.has('....') return` kind of vibes) - that should just work™️ if we invert the process_workflow_engine flag.

Reply:
yes for when 1 processor is enabled, return also works